tox.ini

```diff
@@ -3,11 +3,30 @@
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 
+# This file is set up for environments of the form:
+#  py{*}-{test,nose}[-coverage]
+# Though the env-list is more restricted so that tox without parameters runs
+# something sensible.
+
 [tox]
-envlist = py27, py3
+envlist = py{27,3}
+
+[testenv:py27]
+commands =
+    coverage run --source=./nmosnode -m unittest discover -s tests
+    coverage annotate
+    coverage report
+deps =
+    coverage
+    mock
+install_command = pip install {opts} {packages}
```

Comment:
You actually don't need this line (since actually just becomes pip install i.e. the default)